style.css

```diff
@@ -488,14 +660,6 @@ button:active {
     height: 550px;
   }
 
-  .card_content ul li {
-    font-weight: 600;
-    font-size: 12px;
-    line-height: 16px;
-    letter-spacing: 0.03em;
-    color: #172b4d;
-  }
-
   /* ABOUT */
   .illustration1 img {
     display: none;
```

Comment:
You did well implementing the changes by the previous reviewer, please while you fixed the correction you didn't implement the buttons for the mobile version to looks similar to  the design you were given. Kindly implement this so your UI would look better 😄 

Figma design :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/1d302709-33c8-4b6d-a61a-826a7d34b481)

Your design :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/4b904409-bb5e-4c85-bc64-8e476818bcb2)